components/Collection.vue

```diff
@@ -188,6 +188,24 @@ export default {
 				return undefined;
 			}
 		},
+		missing() {
+			if (Utils.isObject(this.stac.summaries) && Array.isArray(this.stac.summaries['federation:missing'])) {
+				return this.stac.summaries['federation:missing'];
+			}
+			else if (Array.isArray(this.stac['federation:missing'])) {
+				return this.stac['federation:missing'];
+			}
+			else {
+				return undefined;
+			}
+		},
+		affectedByMissing() {
+			if (this.missing && !Array.isArray(this.supportedBy)) {   // in case missing is set but supportedBy is unexpectedly not an array...
+				return true;   // default to display the notice (don't hold back information when we can't be sure)
+			} else {   // otherwise: check if any of the missing backends is actually in the list of backends that are relevant here
+				return this.missing && Array.isArray(this.missing) && this.missing.some(backend => this.supportedBy.includes(backend));
```

Comment:
```suggestion
				return Array.isArray(this.missing) && this.missing.some(backend => this.supportedBy.includes(backend));
```

see https://github.com/Open-EO/openeo-vue-components/pull/102/files#r2036266968